techeria_controller/templates/smartphone.html

```diff
@@ -0,0 +1,205 @@
+
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <div class="d-flex ml-auto flex-column flex-lg-row align-items-center">
+              <ul class="navbar-nav  ">
+                <li class="nav-item active">
+                    <a class="nav-link" href=" {% url 'index' %}  ">Home <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href=" {% static 'about.html' %} "> About </a>
+                </li>
+
+                <li class="nav-item">
+                  <a class="nav-link" href=" {% static 'contact.html' %} ">Contact us</a>
+                </li>
+              </ul>
+              <form class="form-inline my-2 my-lg-0 ml-0 ml-lg-4 mb-3 mb-lg-0">
+                <button class="btn  my-2 my-sm-0 nav_search-btn" type="submit"></button>
+              </form>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <!-- end header section -->
+
+  </div>
+  <div class="bg">
+
+    <h3 class="my-4 border-bottom pb-1"><span class="text-muted">Search Results for </span>{{request.GET.q}}</h3>
+      <div class="container bcontent">
+          {% for q in smartphone%}
+          <hr />
+          <div class="card" style="width: 500px;">
+              <div class="row no-gutters">
+                  <div class="col-sm-5">
+                      <img class="card-img-top" src="/media/smartphones/{{q.image}}" alt="Card image cap">
+                  </div>
+                  <div class="col-sm-7">
+                      <div class="card-body">
+                          <h5 class="card-title">{{ q.name }}</h5>
+                          <p class="card-text">${{ q.price }}</p>
+                          <a href="#" class="btn btn-primary">Details</a>
+                      </div>
+                  </div>
+              </div>
+          </div>
+          {% endfor %}
+      </div>
+
+
+
+
+
+<!-- info section -->
+<section class="info_section layout_padding">
+ <div class="container">
+   <div class="info_items">
+     <a href=" {% static '' %} ">
+       <div class="item ">
+         <div class="img-box box-1">
+           <img src=" {% static '' %} " alt="">
+         </div>
+         <div class="detail-box">
+           <p>
+             Lorem Ipsum is simply dummy text
+           </p>
+         </div>
+       </div>
+     </a>
+     <a href=" {% static '' %} ">
+       <div class="item ">
+         <div class="img-box box-2">
+           <img src=" {% static '' %} " alt="">
+         </div>
+         <div class="detail-box">
+           <p>
+             +02 1234567890
+           </p>
+         </div>
+       </div>
+     </a>
+     <a href=" {% static '' %} ">
+       <div class="item ">
+         <div class="img-box box-3">
+           <img src=" {% static '' %} " alt="">
+         </div>
+         <div class="detail-box">
+           <p>
+             demo@gmail.com
```

Comment:
If we do decide to keep this section, this should be a valid email that belongs to Techeria like techeriawebsite@gmail.com or something similar. And also a valid phone number should be added, so that people can call our Techeria company for any issues that they have with our website.